### DISCUSSION TITLE
problem with the retroarch-metal cask sha256

### DISCUSSION THREAD

**Discussion by MetalGooseSolid** [Homebrew/discussions#3189] [Casks]
I'm not sure where to report this, but the expected SHA256 is incorrect for current version of this cask, preventing installation. It was incorrect for the prior version as well. I'm guessing the SHA256 is being pulled from the wrong place. I tried to find a matching file, but couldn't. The actual SHA256 is correct.

Error: retroarch-metal: SHA256 mismatch
Expected: a897908c5228d9d7487683ef1c234638ce7a5994e5cf9e19d80a2fba745482d7
Actual: 255c109553a63f6dee4b6340386635e3fc8986d07e48b5f2d8599059908fd616

**Comment by SMillerDev** [comment]
https://docs.brew.sh/Common-Issues#cask---checksum-does-not-match

**Comment by Alkaline-Dreams** [comment]
FYI a pull request was opened here: https://github.com/Homebrew/homebrew-cask/pull/122075
